docs/api/configureStore.mdx

```diff
@@ -114,6 +114,91 @@ If the DevTools are enabled by passing `true` or an object, then `configureStore
 
 An optional initial state value to be passed to the Redux `createStore` function.
 
+This allows you to override part (or all) of the initial state values that you have defined for each of your reducers and slices. This is useful for things like hydrating the state from `localStorage` or for [writing tests](https://redux.js.org/usage/writing-tests) for your Redux applications.
+
+> **Note**: `preloadedState` is not recursively partial! For each part of your application's initial state that you decide to provide initial values for, you must define values for **all** properties of that part of state. They will be passed to your reducers as the initial state, and they themselves can't be partial.
+
+Let's discuss the above point some more and provide a few tips for how you fully leverage `preloadedState` in your application and tests. For example, let's stay we start with the following initial state:
```

Comment:
:hand: The info from here on down is useful, but I don't think it needs to be part of the main flow of the page.

Over in the Redux core docs, we have a `<DetailedExplanation>` component that toggles display of content, as soon at https://redux.js.org/style-guide/#structure-files-as-feature-folders-with-single-file-logic and other spots.

We don't have that component in this repo, but it should be straightforward to copy-paste it over and wrap that around all the content after the "Note" block.

Reply:
Perfect! I'll look into adding that. Thank you